msgpack-core/src/main/java/org/msgpack/core/buffer/MessageBuffer.java

```diff
@@ -33,6 +33,19 @@
 
     static {
         try {
+            // Check java version
+            String javaVersion = System.getProperty("java.specification.version", "");
+            int dotPos = javaVersion.indexOf('.');
+            boolean isJavaAtLeast7 = false;
+            if(dotPos == -1) {
+                isJavaAtLeast7 = false;
+            }
+            else {
+                int major = Integer.parseInt(javaVersion.substring(0, dotPos));
```

Comment:
We don't need to catch an exception from `parseInt()` or `substring()` just in case?